### ISSUE TITLE
Change section heading "Carrying the banner"

### ISSUE THREAD

**Issue by tobyhodges** [carpentries-incubator/jekyll-pages-novice#152]
This section heading is not very informative to someone looking for a particular section e.g. with a screen reader. It should be changed to something more descriptive of the actual content of the section e.g. "Adding a site banner"
